pwndbg/ui.py

```diff
@@ -22,13 +23,17 @@
 
 def banner(title):
     title = title.upper()
-    try:
-        _height, width = struct.unpack('hh', fcntl.ioctl(sys.stdin.fileno(), termios.TIOCGWINSZ, '1234'))
-    except:
-        width = 80
+    _height, width = get_window_size()
     width -= 2
     return C.banner(("[{:%s^%ss}]" % (config.banner_separator, width)).format(title))
 
 def addrsz(address):
     address = int(address) & pwndbg.arch.ptrmask
     return "%{}x".format(2*pwndbg.arch.ptrsize) % address
+
+def get_window_size():
+    try:
+        rows, cols = struct.unpack('hh', fcntl.ioctl(sys.stdin.fileno(), termios.TIOCGWINSZ, '1234'))
+    except:
+        rows, cols = (os.environ['LINES'], os.environ['COLUMNS'])
```

Comment:
What is the case when the `except` happens?
Also if someone has no `LINES` or `COLUMNS` env vars it might be better to use `os.environ.get(envvar, default_value)`.